src/python/doc/conf.py

```diff
@@ -19,6 +19,8 @@
     "sphinx_paramlinks",
 ]
 
+autodoc_class_signature = "separated"
```

Comment:
I didn't know this feature, and I prefer it this way (arguments are described only in constructor) !

Reply:
Argh... it creates a `__new__` entry for `*Complex` (but not `WeightedRipsComplex`, nor `DTMRipsComplex`) and `SimplexTree` classes, don't know why. :thinking: 

![image](https://user-images.githubusercontent.com/10407034/228773077-ee678c20-2a25-4662-adc0-14bbf9a5a293.png)

Reply:
Ok, thanks, I did not know this cython behaviour